cli/cmd/inject_test.go

```diff
@@ -18,6 +18,8 @@ func TestInjectYAML(t *testing.T) {
 		{"inject_emojivoto_deployment.input.yml", "inject_emojivoto_deployment.golden.yml"},
 		{"inject_emojivoto_deployment_hostNetwork_false.input.yml", "inject_emojivoto_deployment_hostNetwork_false.golden.yml"},
 		{"inject_emojivoto_deployment_hostNetwork_true.input.yml", "inject_emojivoto_deployment_hostNetwork_true.golden.yml"},
+		{"inject_emojivoto_deployment_controller_name.input.yml", "inject_emojivoto_deployment_controller_name.golden.yml"},
+		{"inject_conduit_deployment_insert_localhost_name.input.yml", "inject_conduit_deployment_insert_localhost_name.golden.yml"},
```

Comment:
NIT: I don't think the `{"inject_conduit_deployment_insert_localhost_name.input.yml", "inject_conduit_deployment_insert_localhost_name.golden.yml"}` test case is needed because we already test that case in install_test.go, so I think we could remove it.

Reply:
Sure I could do that. I wasn't sure if it was better to have the additional test for `inject.go` to specifically highlight this change, considering the fact that the localhost modification happens in `inject.go` rather than `install.go`.